Source/Scene/ClassificationPrimitive.js

```diff
@@ -0,0 +1,916 @@
+     * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready. If false initializeTerrainHeights() must be called first.
+     * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Determines if this primitive's commands' bounding spheres are shown.
+     * @param {Boolean} [options.debugShowShadowVolume=false] For debugging only. Determines if the shadow volume for each geometry in the primitive is drawn. Must be <code>true</code> on
+     *                  creation for the volumes to be created before the geometry is released or options.releaseGeometryInstance must be <code>false</code>.
+     *
+     * @see Primitive
+     * @see GroundPrimitive
+     * @see GeometryInstance
+     * @see Appearance
+     */
+    function ClassificationPrimitive(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        /**
+         * The geometry instance rendered with this primitive.  This may
+         * be <code>undefined</code> if <code>options.releaseGeometryInstances</code>
+         * is <code>true</code> when the primitive is constructed.
+         * <p>
+         * Changing this property after the primitive is rendered has no effect.
+         * </p>
+         * <p>
+         * Because of the rendering technique used, all geometry instances must be the same color.
+         * If there is an instance with a differing color, a <code>DeveloperError</code> will be thrown
+         * on the first attempt to render.
+         * </p>
+         *
+         * @type {Array|GeometryInstance}
+         *
+         * @default undefined
+         */
+        this.geometryInstances = options.geometryInstances;
+        /**
+         * Determines if the primitive will be shown.  This affects all geometry
+         * instances in the primitive.
+         *
+         * @type {Boolean}
+         *
+         * @default true
+         */
+        this.show = defaultValue(options.show, true);
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the bounding sphere for each draw command in the primitive.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the shadow volume for each geometry in the primitive.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugShowShadowVolume = defaultValue(options.debugShowShadowVolume, false);
+        this._debugShowShadowVolume = false;
+
+        this._extruded = defaultValue(options._extruded, false);
```

Comment:
Perhaps add a comment that this is used by the `GroundPrimitive`.  At first glance, I thought this was just a leftover from copy-paste.